### PR TITLE
Update notebooks and docs with updated Cluster Configuration args

### PR DESCRIPTION
**Pull request by Ygnas**
# Issue link

[https://issues.redhat.com/browse/RHOAIENG-7686](https://issues.redhat.com/browse/RHOAIENG-7686)
# What changes have been made

Update notebooks and docs with updated Cluster Configuration args
# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

**Comment by sutaakar** [comment]
@Ygnas Can you please also patch the replacing commands in https://github.com/project-codeflare/codeflare-sdk/blob/main/.github/workflows/guided_notebook_tests.yaml ?

File: docs/cluster-configuration.md [hunk 1]
```diff
@@ -11,14 +11,13 @@ cluster = Cluster(ClusterConfiguration(
     namespace='default', # Default None
     head_cpus=1, # Default 2
     head_memory=1, # Default 8
-    head_gpus=0, # Default 0
-    num_gpus=0, # Default 0
+    head_extended_resource_requests={'nvidia.com/gpu':0}, # Default 0
```
Comment (Bobbins228) [inline docs/cluster-configuration.md:14]:
```suggestion
    head_extended_resource_requests={'nvidia.com/gpu':0}, # Default {'nvidia.com/gpu': 0}
```
Maybe to be as clear as possible we should specify that the default is a `nvidia.com/gpu`

File: docs/cluster-configuration.md [hunk 2]
```diff
@@ -11,14 +11,13 @@ cluster = Cluster(ClusterConfiguration(
...
+    worker_extended_resource_requests={'nvidia.com/gpu':0}, # Default 0
```
Comment (Bobbins228) [inline docs/cluster-configuration.md:15]:
```suggestion
    worker_extended_resource_requests={'nvidia.com/gpu':0}, # Default {'nvidia.com/gpu': 0}
```

**Review by Bobbins228 — CHANGES_REQUESTED** [review]
Small nit then I'm happy with lgtming this

**Review by Bobbins228 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**